### PR TITLE
feat: .vscode

### PR DESCRIPTION
**Pull request by TonybynMp4**
second time's the charm

### CODE REVIEW

File: repo-template/.vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "recommendations": [
+        "sumneko.lua",
+        "overextended.cfxlua-vscode",
+        "ihyajb.qbcore-code-snippets",
```
Comment (D4isDAVID) [inline repo-template/.vscode/extensions.json:5]: Do we still need these snippets? lol

**Review by D4isDAVID — APPROVED** [review]

File: repo-template/.vscode/extensions.json (hunk 1 above)
Comment (TonybynMp4) [reply repo-template/.vscode/extensions.json:5]: I think they're still helpful sometimes 🤔

File: repo-template/.vscode/settings.json
Comment (solareon) [inline repo-template/.vscode/settings.json:1]: Do we want to add qbx to the globals list?

Comment (TonybynMp4) [reply repo-template/.vscode/settings.json:1]: 🤷🏻

Comment (mafewtm) [reply repo-template/.vscode/settings.json:1]: [heres what should be in the globals](https://github.com/Qbox-project/qbx_ambulancejob/blob/main/.vscode/settings.json)

File: repo-template/.vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        "EditorConfig.EditorConfig",
+        "eamodio.gitlens",
+        "GitHub.vscode-pull-request-github"
```
Comment (Manason) [inline repo-template/.vscode/extensions.json]: What are these three? They sound like they may be a personal set preference and therefore maybe shouldn't be on here.

**Review by Manason — CHANGES_REQUESTED** [review]

File: repo-template/.vscode/extensions.json (hunk 2 above)
Comment (D4isDAVID) [reply repo-template/.vscode/extensions.json]: Recently a `.editorconfig` was added to all repos so we should keep it. GitLens is pretty good for any git-related features, but I guess it's optional. The pull request one can probably be removed.

Comment (TonybynMp4) [reply repo-template/.vscode/extensions.json]: I copied straight from the core so idk i thought it was something agreed upon already 🤷🏻

Comment (D4isDAVID) [reply repo-template/.vscode/extensions.json]: Nah it's just random stuff I added back in QBCore

Comment (Manason) [reply repo-template/.vscode/extensions.json]: I think we should just remove pull request and gitlens. I use git on command line and many others use the GitHub application directly.

**Review by Manason — APPROVED** [review]

**Pull request merged**